src/components/ex-accordion/ex-accordion.scss

```diff
@@ -0,0 +1,52 @@
+.accordion {
+  @apply mt-component-spacing-small;
+
+  .cmp-accordion__item {
+    @apply mb-2;
+
+    background-color: #f5f8fa;
+
+    &:last-child {
+      @apply mb-0;
+    }
+
+    .cmp-accordion__header {
+      @apply font-headline text-default text-gray-1 px-8 py-3 font-semibold;
+
+      &:hover {
+        @apply text-primary;
+      }
+
+      .cmp-accordion__button {
+        @apply flex items-center;
+
+        &:focus {
+          outline: none;
+        }
+
+        &::before {
+          @apply inline-block mr-3 bg-center bg-no-repeat bg-contain;
+          transition: 0.2s transform;
+          content: '';
+          width: 15px;
+          height: 20px;
+          background-image: url('./resources/images/sicherearbeit-arrow-gray.svg');
```

Comment:
We should not make the same mistake like on ÖGB. The arrow should point downwards to indicate there is something to expand - and point upwards to indicate there is something to collapse.

Reply:
Changed the direction of the arrow